### DISCUSSION TITLE
Endpoint search will cause the entire page to jam

### DISCUSSION THREAD

**Discussion by Mxgxg** [apache/skywalking#7514] [Q&A]
In version 8.7, after selecting the service, enter the endpoint name, which cannot be selected, and the whole page cannot be clicked and refreshed. Is this a bug or my configuration problem？
![1629451127](https://user-images.githubusercontent.com/79822553/130211441-46f1735c-963d-4987-b2c0-c71a646d953a.jpg)

**Comment by wu-sheng** [comment]
This depends, does backend have timeout?

**Reply by Mxgxg** [reply]
There are no exceptions on the backend

**Reply by wu-sheng** [reply]
Then, I have no clue. I didn't face this before.

**Reply by Mxgxg** [reply]
ok

**Reply by Mxgxg** [reply]
I hope you can pay attention. It feels like a bug

**Reply by wu-sheng** [reply]
Your ask is wrong. This is a community. If something is not working, it is common. If someone wants to fix, do it. There is nothing we pay attention. This is not the responsibility for any single person.

**Reply by git1e** [reply]
I met the same problem as you, how did you solve it ?@Mxgxg

**Reply by Mxgxg** [reply]
> I met the same problem as you, how did you solve it ?@Mxgxg

version 8.8 is fixed

**Comment by wu-sheng** [comment]
I double checked, there is no same issue on the official demo env. Nothing wrong. It is 8.7.0 deployment.

**Comment by wu-sheng** [comment]
@Fine0830 I accidentally find, that endpoint list seems missing backend search for trace and log pages. Could you help on recheck?
